metabase/metabase
# 42592 new modal adding database

Closes https://github.com/metabase/metabase/issues/42592

### Description
This PR removes the old "Explore Sample Data" modal when adding a new database for the first time, with a modal that prompts you to go set permissions on the tables. 

### How to verify
1. Admin -> Database -> Add a new Database
2. After hitting Save, you should be redirected to the Database List page with a modal prompting you to add permissions to the tables.

### Demo
![chrome_pmyRJCOznr](https://github.com/metabase/metabase/assets/1328979/9ce9cbc6-6067-4951-92c8-0bdda58d6994)

### Checklist
